### PR TITLE
[Discipline Priest] Fix for the T20 2pc Nerf

### PR DESCRIPTION
**Pull request by karlpralow**
Fix for the T20 2pc Nerf. This also includes the healing being calculated on the first bolt since Blizzard fixed it

### CODE REVIEW

**Comment by karlpralow** [comment]
It's in the CombatLogParser. I'll move it while I'm working on this

File: src/Parser/Priest/Discipline/Modules/Items/Tier20_2set.js [hunk 1]
```diff
@@ -23,26 +23,26 @@ class Tier20_2set extends Analyzer {
...
-      this._secondPenanceBoltLastDamageEvent = false;
+    if (event.ability.guid !== SPELLS.PENANCE.id || event.penanceBoltNumber !== 0) {
+      this._firstPenanceBoltLastDamageEvent = false;
       return;
     }
 
-    this._secondPenanceBoltLastDamageEvent = true;
-    this.damage += (event.amount / 2);
+    this._firstPenanceBoltLastDamageEvent = true;
+    this.damage += (event.amount / 1 + TIER_20_TWO_SET_BONUS);
```
[the remark this answers is not in the document]
Comment (karlpralow) [reply src/Parser/Priest/Discipline/Modules/Items/Tier20_2set.js]: I found some in each damage classes (getDamageDone) that is copied in each class. Is there one in core that is shared or should I create it ?

[the remark this answers is not in the document]
Comment (kfinch) [reply src/Parser/Priest/Discipline/Modules/Items/Tier20_2set.js]: There should be one in core, go ahead and make it.

[the remark this answers is not in the document]
Comment (kfinch) [reply src/Parser/Priest/Discipline/Modules/Items/Tier20_2set.js]: Sorry to be clear I mean "there isn't one in core but there should be"

**Comment by kfinch** [comment]
Looks good. Are you ready to merge, or did you want to move the `item()` in first?

**Comment by karlpralow** [comment]
It will do another PR for the item()

**Pull request merged**